siunitx.dtx

```diff
@@ -17840,7 +17842,7 @@ This work consists of the file  siunitx.dtx
 %    \end{macrocode}
 % \end{macro}
 % \begin{macro}
-%   {\W, \uW, \mW, \kW,\MW , \GW, \kJ, \J, \eV, \meV, \keV, \MeV, \GeV, \TeV, \kWh}
+%   {\W, \uW, \mW, \kW, \MW , \GW, \kJ, \J, \mJ, \uJ \eV, \meV, \keV, \MeV, \GeV, \TeV, \kWh}
```

Comment:
The added space before \MW should have been a moved one, there is a supplemental space right after it.